01-arch/07-reputation.md

```diff
@@ -0,0 +1,41 @@
+## Reputation system
+
+\Gls{NeoFS} reputation system is a subsystem for calculating trust in \Gls{Node}. It is based on a reputation model for assessing trust, which, in turn, is based on the EigenTrust algorithm designed for peer-to-peer reputation management. The algorithm implies that there is a uniquely defined manager (parent node) for each network participant at each fixed moment of time (epoch). The manager iteratively determines a complex general (global) trust of the entire network to the child node based on information received from its child node and other managers.
+
+### Trust
+
+Trust in a \Gls{Node} in \Gls{NeoFS} is its quantitative (numerical) assessment based on the experience of interacting with that \Gls{Node}. The higher the score -- the higher the trust in the node and vice versa. Since the system uses a reputation-based trust model, the terms “trust” and “reputation” are considered synonymous in this documentation.
```

Comment:
... in a \Gls{NeoFS} \Gls{Node}...
The higher the score is, the higher is the trust in the node and vice versa.